src/containers/ConversionForm/index.tsx

```diff
@@ -36,6 +38,13 @@ interface ConversionProps {
     setError: ActionCreator<SetErrorAction>;
 }
 
+const validFileExtensions: ExtensionMap = {
+    Smoldyn: "txt",
+    cytosim: "txt",
+    cellPACK: "txt",
+    SpringSaLaD: "txt",
```

Comment:
put a TODO here to update these with their correct format once we support them, so that we know this isn't necessarily the correct data